tests/integration/cli/test_schema_records_validation.py

```diff
@@ -0,0 +1,132 @@
+    assert 'Schema not found.' in res.output
+
+
+def test_validate_with_ana_type_with_invalid_ana_version(app, db, cli_runner, script_info, create_schema):
+    create_schema('test', experiment='CMS')
+    res = cli_runner.run(
+        'fixtures validate -a test -v version', obj=script_info)
+
+    assert res.exit_code == 2
+    assert 'Version has to be passed as string <major>.<minor>.<patch>.' in res.output
+
+
+def test_validate_with_schema_url_with_invalid_url(app, cli_runner, script_info):
+    res = cli_runner.run(
+        'fixtures validate -u invalid-schema-url', obj=script_info)
+
+    assert res.exit_code == 2
+    assert 'Please provide a valid schema url' in res.output
+
+
+def test_validate_with_schema_url_schema_doesnt_exist(app, db, cli_runner, script_info):
+    res = cli_runner.run(
+        'fixtures validate -u https://analysispreservation.cern.ch/schemas'
+        '/deposits/records/invalid-schema-v0.0.1.json', obj=script_info)
+
+    assert res.exit_code == 2
+    assert 'Schema not found.' in res.output
+
+
+def test_validate_with_schema_url_version_doesnt_exist(app, db, cli_runner, script_info, create_schema):
+    create_schema('test', experiment='CMS')
+    res = cli_runner.run(
+        'fixtures validate -u https://analysispreservation.cern.ch/schemas'
+        '/deposits/records/test-v0.0.2.json', obj=script_info)
+
+    assert res.exit_code == 2
+    assert 'Schema not found.' in res.output
+
+
+def test_validate_with_schema_url_correct_record(
+        app, db, es, cli_runner, script_info, users, create_schema, create_deposit):
+    create_schema('test', experiment='CMS')
+    create_deposit(users['cms_user'], 'test')
+
+    res = cli_runner.run(
+        'fixtures validate -u https://analysispreservation.cern.ch/schemas'
+        '/deposits/records/test-v1.0.0.json', obj=script_info)
+
+    assert res.exit_code == 0
+    assert '1 record(s) of test found.' in res.output
+    assert 'No errors found in record' in res.output
+
+
+def test_validate_with_ana_type_correct_record(
+        app, db, es, cli_runner, script_info, users, create_schema, create_deposit):
+    create_schema('test', experiment='CMS')
+    create_deposit(users['cms_user'], 'test')
+
+    res = cli_runner.run('fixtures validate -a test', obj=script_info)
+
+    assert res.exit_code == 0
+    assert '1 record(s) of test found.' in res.output
+    assert 'No errors found in record' in res.output
+
+
+def test_validate_with_schema_url_wrong_record(
+        app, db, es, cli_runner, script_info, users, create_schema, create_deposit):
+    create_schema('test', experiment='CMS')
+    create_deposit(users['cms_user'], 'test')
+
+    res = cli_runner.run(
+        'fixtures validate -u https://analysispreservation.cern.ch/schemas'
+        '/deposits/records/test-v1.0.0.json', obj=script_info)
+
+    assert res.exit_code == 0
+    assert '1 record(s) of schema test found.' in res.output
+    assert 'No errors found in record' in res.output
```

Comment:
- [ ] Add test that check returned errors
- [ ] Add test that check returned errors and creates output